api/src/main/java/org/openmrs/Concept.java

```diff
@@ -1419,7 +1419,7 @@ public String toString() {
 			}
 		}
 		catch (Exception e) {
-			// pass
+		  throw new RuntimeException(e);
```

Comment:
Did you use our code formatter? https://wiki.openmrs.org/display/docs/Java+Conventions

Reply:
done